pkg/system/utils_unix.go

```diff
@@ -26,7 +27,8 @@ func resolveBinary(binname string) (string, error) {
 	return "", fmt.Errorf("Binary %q does not resolve to a binary of that name in $PATH (%q)", binname, resolvedPath)
 }
 
-func execCmd(cmd, args string) ([]byte, error) {
+// ExecCmd calls exec.Command and returns the CombinedOutput.
+func ExecCmd(cmd, args string) ([]byte, error) {
```

Comment:
I don't think we should export this two line helper.

I believe with the suggested split up  this won't need to be exported anymore, as the only consumers will be in `pkg/system/user`